patches/@types+jsdom++parse5+7.0.0.patch

```diff
@@ -0,0 +1,91 @@
+diff --git a/node_modules/@types/jsdom/node_modules/parse5/dist/index.d.ts b/node_modules/@types/jsdom/node_modules/parse5/dist/index.d.ts
+index 81253d3..d2333bf 100644
+--- a/node_modules/@types/jsdom/node_modules/parse5/dist/index.d.ts
++++ b/node_modules/@types/jsdom/node_modules/parse5/dist/index.d.ts
+@@ -1,10 +1,10 @@
+-import { type ParserOptions } from './parser/index.js';
++import { ParserOptions } from './parser/index.js';
```

Comment:
Sorry why do we have to remove `type` here?

Reply:
Ohh... wow. That's subtle. How do other people even use this package then 🤔

Reply:
no idea. unless it has something to do with our setup.